crates/oxc_semantic/src/node.rs

```diff
@@ -234,6 +241,7 @@ impl<'a> AstNodes<'a> {
         );
         self.parent_ids.push(NodeId::ROOT);
         self.nodes.push(AstNode::new(kind, scope_id, cfg_id, flags, NodeId::ROOT));
+        self.node_kinds_set.set(kind.ty());
```

Comment:
```suggestion
        self.node_kinds_set.set(AstKind::Program);
```

Or, since every AST contains `Program`, could just initialize `AstTypesBitset` at the start with that bit already set.

Reply:
> Or, since every AST contains `Program`, could just initialize `AstTypesBitset` at the start with that bit already set.

Yeah this did occur to me. Maybe we'll do it in a follow-up PR, it's probably very minor at the moment.